usaspending_api/database_scripts/etl/transaction_delta_view.sql

```diff
@@ -341,7 +344,8 @@ LEFT JOIN (
  WHERE
    faba.award_id IS NOT NULL
  GROUP BY
-   faba.award_id
+   faba.award_id,
+   faba.linked_date
```

Comment:
Seems problematic to group by `linked_date`. Would obtaining the max be a viable alternative?

Reply:
I agree. Switching over to the MAX would be better